tutorials/colors/colormap-manipulation.py

```diff
@@ -25,18 +25,18 @@
 
 First, getting a named colormap, most of which are listed in
 :doc:`/tutorials/colors/colormaps`, may be done using
-`.matplotlib.cm.get_cmap`, which returns a colormap object.
+`.matplotlib.colormaps`, which returns a colormap object.
 The second argument gives the size of the list of colors used to define the
 colormap, and below we use a modest value of 8 so there are not a lot of
 values to look at.
```

Comment:
There's no second argument to a dictionary access. Should change to talk about `resample`.